### ISSUE TITLE
ClassCastException when using JBOSS Docker Plugin and Doclipser

### ISSUE THREAD

**Issue by MUI-Pop** [domeide/doclipser#25]
Getting ClassCastException when using JBOSS Docker Plugin and Doclipser. I see this because both using same Docker client library 'Spotify'.

**Comment by rgrunber** [comment]
Yes, this is likely happening because there are 2 instances of spotify/docker-client. One provided by JBOSS Docker Plugin (from linuxtools.docker) as an OSGi bundle, and one that is bundled by com.zenika.doclipser.api. You might end up seeing something like "org.foo.Bar cannot be cast to org.foo.Bar".

I guess one possibility would be to use http://download.eclipse.org/tools/orbit/downloads/drops/R20150821153341/ which contains a p2 repo of docker-client 3.1.1.
